projects/angular/src/layout/breadcrumbs/breadcrumbs.html

```diff
@@ -0,0 +1,34 @@
+<div role="list" class="clr-breadcrumb-menu">
+  <ng-container *ngIf="items?.length">
+    <clr-breadcrumb-item *ngIf="items.length > max && !isExpanded">
+      <button class="btn btn-link btn-sm clr-breadcrumb-expand" (click)="expand()" (keydown)="expand()">
+        <cds-icon shape="ellipsis-horizontal"></cds-icon>
+      </button>
+    </clr-breadcrumb-item>
+    <ng-container *ngFor="let breadcrumb of items | slice: -limit : items.length; let isLastItem=last">
+      <clr-breadcrumb-item>
```

Comment:
```suggestion
    <clr-breadcrumb-item *ngFor="let breadcrumb of items | slice: -limit : items.length; let isLastItem=last">
```

no need for an `ng-container` here.